File: Yeni klasör (3)/12111/WindowsFormsApplication2/WindowsFormsApplication2/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.IO;
namespace WindowsFormsApplication2
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
    struct   bilgiler 
       {
        public string adi;
        public string soyadi;
       }
    Int32 toplam;
        double ortalama=0;
        private void button1_Click(object sender, EventArgs e)

        {
            StreamWriter yaz;
            yaz = File.AppendText("bilgiler.txt");
            yaz.WriteLine(textBox1.Text );
            yaz.Close();


        }
        DialogResult sor;
        private void Form1_Load(object sender, EventArgs e)
        {
            if (File.Exists("bilgiler.txt") == true)
            {
                MessageBox.Show("dosya mevcut");
            }
            else
            {
                sor = MessageBox.Show("Dosya oluşturulsunmu","",MessageBoxButtons.YesNo,MessageBoxIcon.Question );
                if (sor == DialogResult.Yes)
                {
                    StreamWriter dosya;
                    dosya = File.CreateText("bilgiler.txt");

                
                }
            
            }
        }
    }
}

File: RAPORLAMA/Form1.vb
Imports System.Data.OleDb
Public Class Form1
    Dim baglanti As New OleDbConnection("Provider=Microsoft.Jet.Oledb.4.0; Data Source=bilgiler.mdb")
    Sub guncelle()
        Dim adaptor As New OleDbDataAdapter("SELECT * FROM TABLO1", baglanti)
        Dim tablo As New DataTable()
        adaptor.Fill(tablo)
        DataGridView1.DataSource = tablo


    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        guncelle()

    End Sub
    Public sql As String = ""

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        sql = "SELECT * FROM TABLO1"
        Form2.ShowDialog()

    End Sub
    Sub temizle()
        TextBox1.Clear()
        TextBox2.Clear()

    End Sub
    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        If (TextBox1.Text <> "") And (TextBox2.Text <> "") Then

            sql = "select * from tablo1 where ADI='" + TextBox1.Text.Trim() + "'AND SOYADI='" + TextBox2.Text.Trim() + "'"
            Dim kaydet As New OleDbCommand(sql, baglanti)
            baglanti.Open()
            Dim varmı As OleDbDataReader = kaydet.ExecuteReader()
            If (varmı.Read()) Then
                Dim sor As Integer

                sor = MessageBox.Show("ARANILAN KAYIT BULUNDU GÖSTERİLSİN Mİ ?", "UYARI", MessageBoxButtons.YesNo, MessageBoxIcon.Question)
                If (sor = DialogResult.Yes) Then
                    Form2.ShowDialog()
                    temizle()
                Else

                    temizle()


                End If

            Else

                MessageBox.Show("KAYIT BULUNAMADI...")



            End If
        guncelle()
        baglanti.Close()

        Else

        MessageBox.Show("müştrei adı veya soyadını gırınız...")

        End If

    End Sub
End Class
